test/duplicates.jl

```diff
@@ -20,7 +20,7 @@ module TestDuplicates
     @test pdf ≅ updf
 
     @testset "missing" begin
-        df = DataFrame(A = 1:12, B = repeat('A':'C', inner=4))
-        @test DataFrames.colmissing(df) == [0, 0]
+        df1 = DataFrame(A = 1:12, B = repeat('A':'C', inner=4))
```

Comment:
I don't think we should change this. There are many more warnings like this which are totally spurious, better just wait until the deprecations are removed.

Reply:
I think the issue is that some of the tests are outside of `@testset` and some are within.  I agree that it is probably not a good idea to fix a few of these in isolation.

Reply:
Yeah, but we really don't care whether the variable is global or local here anyway since we generally (always?) overwrite it.